packages/theme/src/cli/services/local-storage.ts

```diff
@@ -64,48 +65,56 @@ export function setThemeStore(store: string) {
 
 export function getDevelopmentTheme(): string | undefined {
   outputDebug(outputContent`Getting development theme...`)
-  return developmentThemeLocalStorage().get(getThemeStore())
+  return developmentThemeLocalStorage().get(requireThemeStore())
 }
 
 export function setDevelopmentTheme(theme: string): void {
   outputDebug(outputContent`Setting development theme...`)
-  developmentThemeLocalStorage().set(getThemeStore(), theme)
+  developmentThemeLocalStorage().set(requireThemeStore(), theme)
 }
 
 export function removeDevelopmentTheme(): void {
   outputDebug(outputContent`Removing development theme...`)
-  developmentThemeLocalStorage().delete(getThemeStore())
+  developmentThemeLocalStorage().delete(requireThemeStore())
 }
 
 export function getREPLTheme(): string | undefined {
   outputDebug(outputContent`Getting REPL theme...`)
-  return replThemeLocalStorage().get(getThemeStore())
+  return replThemeLocalStorage().get(requireThemeStore())
 }
 
 export function setREPLTheme(theme: string): void {
   outputDebug(outputContent`Setting REPL theme to ${theme}...`)
-  replThemeLocalStorage().set(getThemeStore(), theme)
+  replThemeLocalStorage().set(requireThemeStore(), theme)
 }
 
 export function removeREPLTheme(): void {
   outputDebug(outputContent`Removing REPL theme...`)
-  replThemeLocalStorage().delete(getThemeStore())
+  replThemeLocalStorage().delete(requireThemeStore())
 }
 
 export function getStorefrontPassword(): string | undefined {
-  const themeStore = getThemeStore()
+  const themeStore = requireThemeStore()
   outputDebug(outputContent`Getting storefront password for shop ${themeStore}...`)
-  return themeStorePasswordStorage().get(getThemeStore())
+  return themeStorePasswordStorage().get(themeStore)
 }
 
 export function setStorefrontPassword(password: string): void {
-  const themeStore = getThemeStore()
+  const themeStore = requireThemeStore()
   outputDebug(outputContent`Setting storefront password for shop ${themeStore}...`)
   themeStorePasswordStorage().set(themeStore, password)
 }
 
 export function removeStorefrontPassword(): void {
-  const themeStore = getThemeStore()
+  const themeStore = requireThemeStore()
   outputDebug(outputContent`Removing storefront password for ${themeStore}...`)
   themeStorePasswordStorage().delete(themeStore)
 }
+
+function requireThemeStore(): string {
+  const themeStore = getThemeStore()
+  if (!themeStore) {
+    throw new AbortError('Theme store is not set')
```

Comment:
2 thoughts about this line:

1. Should this be an `AbortError` or a `BugError`? This isn't really an anticipated condition, we shouldn't ever get here, so maybe `BugError` is more appropriate here, because it indicates we should fix the code? I believe only a `BugError` will trigger our monitoring.
2. I went very basic on this message, but it isn't very helpful to the user. What about something like this:
    ```
    Theme store is not set. If your command has a `--store` flag, try running again with
    an explicit value for that flag.
    ```

Reply:
This message is great! Thanks @amcaplan